### ISSUE TITLE
Can we have more space between each title on join dialog?

### ISSUE THREAD

**Issue by neiriod** [drawpile/Drawpile#747]
in latest version
![Snipaste_2019-04-15_20-59-38](https://user-images.githubusercontent.com/49644585/56138320-41e36e80-5fc9-11e9-87cb-dd8e90890be2.png)
                                       
in older version
![Snipaste_2019-04-15_21-01-45](https://user-images.githubusercontent.com/49644585/56138330-44de5f00-5fc9-11e9-8c2f-8600a32c7631.png)

In latest version, it's kinda crowded between each title, which making the information too focused and hard to pick up from eyes.
Can you possibly make it have more space between each title , just the way like in older version?

**Comment by liziaa** [comment]
Agreed, I would prefer have some vertical space between each title.

**Comment by Wade821** [comment]
I would like to add, if possible, that this padding between listings be configurable based on user preference. I personally like the more compressed layout as this means less scrolling to see all the sessions on a higher resolution scree.
